clang/docs/ReleaseNotes.rst

```diff
@@ -606,6 +606,8 @@ Attribute Changes in Clang
 
 - Clang now disallows the use of attributes after the namespace name. (#GH121407)
 
+- Clang now raises warnings for unknown namespaced attributes only in pedantic mode (#GH120875).
```

Comment:
This is incorrect; it adds a new diagnostic group rather than a pedantic warning.